packages/botonic-plugin-knowledge-bases/src/hubtype-knowledge-api-service.ts

```diff
@@ -19,7 +19,7 @@ export class HubtypeApiService {
     chatId: string
   ): Promise<
     AxiosResponse<{
-      ai: string
+      answer: string
       has_knowledge: boolean
       sources: {
         knowledge_source_id: string
```

Comment:
Doubt: Why not creating a `type` or `interface` with the API response the same way it's done in the Flow Builder plugin? 
In fact, I think that the interface should be declared and exported in the Knowledge base plugin and used it in the Flow Builder plugin.

Reply:
I have not created this interface because it is in snake_case and it is only typed here like this because when you use the plugin everything is written in camelCase and does the change internally.
You can't do this import either because the  plugin-flow-builder does not have the plugin-knowledge-bases dependency.